extensions/amp-list/0.1/amp-list.js

```diff
@@ -74,9 +87,32 @@ export class AmpList extends AMP.BaseElement {
     /** @const @private {string} */
     this.initialSrc_ = element.getAttribute('src');
 
+    /** @private @const {boolean} */
+    this.loadMoreEnabled_ =
+        isExperimentOn(this.getAmpDoc().win, LOAD_MORE_EXP_TAG) &&
+        element.hasAttribute('load-more');
+
+    /** @private {?string} */
+    this.loadMoreSrc_ = null;
+
+    /** @private {?Element|undefined} */
+    this.loadMoreOverflow_ = undefined;
```

Comment:
we normally do `null` instead of `undefined` as default then type can be just  `{?Element}`